embedded/projects/SH-Xpresso-LPC54102/sources/app/sensoracq_t.c

```diff
@@ -219,19 +224,19 @@ void SensorControlCmdHandler(MsgSensorControlData *pData)
 #ifdef SENSOR_CONTROL_ENABLE
             // Maybe add a user configurable 'powerMode' setting and here we
             // configure the sensor hardware according to its value.
-            Accel_ConfigDataInt(false);         // do not disable accel so watch window input will work
+            Driver_Acc.Activate(false,0,0,0);         // do not disable accel so watch window input will work
```

Comment:
I had misunderstood this argument since there was no function header. Replacing it with correct macro.